dockerfy/dockers/back/Dockerfile

```diff
@@ -12,9 +14,23 @@ RUN  apt-get update \
     libmojolicious-plugin-i18n-perl libdbd-sqlite3-perl debconf adduser libdigest-sha-perl libnet-ssh2-perl \
     libfile-rsync-perl libdate-calc-perl libparallel-forkmanager-perl libdatetime-perl libencode-locale-perl netcat-openbsd \
     liblwp-useragent-determined-perl libvirt-clients supervisor net-tools openssh-client apt-utils curl libpbkdf2-tiny-perl \
-    libio-stringy-perl libvirt-daemon-system libvirt-clients netcat-openbsd qemu-kvm qemu-utils iproute2 wget bridge-utils firewalld dnsmasq iptables ebtables tzdata \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
+    libio-stringy-perl libvirt-daemon-system libvirt-clients netcat-openbsd qemu-kvm qemu-utils iproute2 wget bridge-utils firewalld dnsmasq iptables ebtables \
+    tzdata unzip \
+ && apt-get clean && \
+   echo "**** download ravada files ****" && \
+   mkdir /ravada && \
+   curl -o \
+   /tmp/ravada.zip -L \
+   "https://github.com/UPC/ravada/archive/${RAVADA_BRANCH}.zip" && \
+   unzip /tmp/ravada.zip -d /tmp/ravadatmp && \
+   cp -r /tmp/ravadatmp/ravada-develop/* /ravada && \
+   cp /ravada/dockerfy/dockers/back/network.sh / && \
+   cp /ravada/dockerfy/dockers/back/default.xml / && \
+   cp /ravada/dockerfy/dockers/back/ravada.conf /etc/ && \
+   cp /ravada/dockerfy/dockers/back/supervisord.conf /etc/ \
+ && rm -rf \ 
+   /var/lib/apt/lists/* \
+   /tmp/*
```

Comment:
I am downloading all files in the github repo for RAVADA_BRANCH set as env variable on line 5, unzipping to a temporary directory and copying everything into /ravada inside the container upon image creation. This avoids the need having to mount the /home/src/ravada directory from host to container (which means you don't have to git clone the repo, you just need the docker-compose.yml and .env files, provided that the DockerHub images get updated to use my code). After that, we are cleaning the /tmp location to get rid of the extra zip and unzipped files.